docs/getting-started-with-scalardb-by-using-kotlin.md

```diff
@@ -198,7 +198,7 @@ To apply the schema, go to the [`scalardb` Releases](https://github.com/scalar-l
 Then, run the following command, replacing `<VERSION>` with the version of the ScalarDB Schema Loader that you downloaded:
 
 ```shell
-$ java -jar scalardb-schema-loader-<VERSION>.jar --config scalardb.properties --schema-file emoney.json --coordinator
+$ java -jar scalardb-schema-loader-<VERSION>.jar --config scalardb.properties --schema-file emoney.json --coordinator --replication-factor=1
```

Comment:
@josh-wong Please review the changes in Java and Kotlin getting started documents when you can.

Ideally, we could display the `--replication-factor=1` option and the note below only when the user has selected Cassandra as the storage, as we do for the previous step, but I am unsure of the best way to do this and if it is worth it.

Reply:
@Torch3333 How about if we use content tabs to show the commands and notice block for each database?

Attached is a video of what that would look like.

https://github.com/scalar-labs/scalardb/assets/23216828/e610fd9c-76db-495d-8bf7-b82729a160da

Reply:
That looks good, thank you. Let's go with your suggestion.

Reply:
I can modify both readme files if you suggest to me how to and you could push changes directly to this PR. Whichever is fine.

Reply:
I've added the code for the content tabs (tested in my local environment) and slightly edited the new content😊

Reply:
Great, thank you.